src/arm7tdmi.rs

```diff
@@ -93,82 +95,113 @@ impl Arm7tdmi {
         todo!("Branch Link")
     }
 
-    fn mov(&mut self, op_code: u32) {
-        // bits [24-21] are the RD
-        let rd = (op_code & 0x00_00_F0_00) >> 12;
-        println!("RD: {:?}", rd);
-
-        // 25th bit is I = Immediate Flag
-        let immediate: bool = (op_code & 0x02_00_00_00) >> 25 == 1;
-        println!("Immediate: {:?}", immediate);
-
-        // 20th bit is S = Condition Set
-        if op_code & 0x00_08_00_00 > 0 {
-            todo!("Condition set")
-        }
-
-        if immediate {
-            // bits [7-0] are the immediate value
-            let immediate_value = op_code & 0x00_00_00_FF;
-            println!("value: {:?}", immediate_value);
+    fn data_processing(&mut self, op_code: u32) {
+        /// bit [25] is I = Immediate Flag
```

Comment:
All these commented lines should be `//`

Reply:
Fixed